themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1183,6 +1183,21 @@ button.tc-btn-invisible.tc-remove-tag-button {
 	color: <<colour muted-foreground>>;
 }
 
+.tc-tiddler-lazy-loading {
+	height: 4px;
+	width: 100%;
+	--c:no-repeat linear-gradient(<<colour background>> 0 0);
```

Comment:
The core does not currently use CSS variables; can we avoid them here?

Reply:
I copied this from https://css-loaders.com/ , I'm not good at writing styles.